metrics/examples/grafana/strimzi-zookeeper.json

```diff
@@ -485,7 +485,7 @@
       "dashLength": 10,
       "dashes": false,
       "datasource": "${DS_PROMETHEUS}",
-      "description": "Kafka Broker Pods Memory Usage",
```

Comment:
Why have you changed the description? This panel displays the memory usage of the zookeeper, not of the kafka brokers.

Reply:
Exactly. That is why I changed it to Zookeeper.